README.md

```diff
@@ -103,7 +103,9 @@ and percentage format ending with `%`), `Float` (also supporting percentage),
 is in this list
 - `Option` for optional values, i.e. value that can or cannot be in the configuration
 - `Map` with `String` keys and any value type that is in this list
-- `LocalDate`, `LocalTime`, `LocalDatetime` (see [Configurable converters](#configurable-converters))
+- everything in [`java.time`](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html).
```

Comment:
Nitpick: It seems like saying "everything" might be a bit of a stretch given types like `Clock` and `MonthDay` aren't covered.